RecoJets/JetProducers/python/pileupjetidproducer_cfi.py

```diff
@@ -22,7 +22,7 @@
                                       runMvas = cms.bool(True),
                                       jets = cms.InputTag("ak4PFJetsCHS"),
                                       vertexes = cms.InputTag("offlinePrimaryVertices"),
```

Comment:
In fact there is yet another file
https://github.com/jbrands/cmssw/blob/newOfflinePuID/RecoJets/JetProducers/python/PileupJetID_cfi.py
that looks like a renamed version of this config that may need this change as well.

Probably (after this PR introducing the new ID) a merge of the various cfi defining the offline pilupjet ID is needed as well:
https://github.com/jbrands/cmssw/blob/newOfflinePuID//PhysicsTools/PatAlgos/python/slimming/pileupJetId_cfi.py
https://github.com/jbrands/cmssw/blob/newOfflinePuID/RecoJets/JetProducers/python/PileupJetID_cfi.py
https://github.com/jbrands/cmssw/blob/newOfflinePuID/RecoJets/JetProducers/python/pileupjetidproducer_cfi.py